db/models.py

```diff
@@ -1 +1,46 @@
 from django.db import models
+
+
+class Race(models.Model):
+    RACE_CHOICE = [
+        ("Elf", "Elf"),
+        ("Dwarf", "Dwarf"),
+        ("Human", "Human"),
+        ("Ork", "Ork")
+    ]
+    name = models.CharField(max_length=255, unique=True, choices=RACE_CHOICE)
+    description = models.TextField(blank=True)
+
+    def __str__(self) -> str:
+        return self.name
+
+
+class Skill(models.Model):
+    name = models.CharField(max_length=255, unique=True)
+    bonus = models.CharField(max_length=255)
```

Comment:
Consider changing the 'bonus' field to a more appropriate field type if it represents a numerical value or a specific set of choices. Currently, it is a CharField, which may not be suitable depending on the intended use.